src/Mod/Part/App/TopoShapePyImp.cpp

```diff
@@ -2069,21 +2069,42 @@ pos=Gui.ActiveDocument.ActiveView.getCameraNode().position.getValue().getValue()
 pos=App.Vector(*pos)
 
 shape=App.ActiveDocument.ActiveObject.Shape
-reflect=shape.reflectLines(ViewDir=vdir, ViewPos=pos, UpDir=udir)
+reflect=shape.reflectLines(ViewDir=vdir, ViewPos=pos, UpDir=udir, EdgeType="Sharp", Visible=True, OnShape=False)
 Part.show(reflect)
  */
 PyObject* TopoShapePy::reflectLines(PyObject *args, PyObject *kwds)
 {
-    static char *kwlist[] = {"ViewDir", "ViewPos", "UpDir", NULL};
+    static char *kwlist[] = {"ViewDir", "ViewPos", "UpDir", "EdgeType", "Visible", "OnShape", NULL};
 
-    PyObject *pView, *pPos, *pUp;
-    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O!O!O!", kwlist,
+    char* type="OutLine";
+    PyObject* vis = Py_True;
+    PyObject* in3d = Py_False;
+    PyObject* pPos = new Base::VectorPy(new Base::Vector3d(0,0,0));

```

Comment:
Here you cannot initialize pPos and pUp with a VectorPy object because it won't be destroyed anymore. Instead you must set them to nullptr and then check below if pPos or pUp are still null or not.

Reply:
OK, I get it. Thanks for the fix.